protos/flyteidl/admin/task.proto

```diff
@@ -51,6 +55,9 @@ message TaskList {
 message TaskSpec {
     // Template of the task that encapsulates all the metadata of the task.
     core.TaskTemplate template = 1;
+
+    // Represents the specification for Description.
+    DescriptionEntity description_entity        = 2;
```

Comment:
Is this always going to be populated in requests to Get and List tasks?

Reply:
TaskSpec is only used for [TaskCreateRequest](https://github.com/flyteorg/flyteidl/blob/cb8df7146b91b8269d1a4ea9b2a1fda42b56b6ff/protos/flyteidl/admin/task.proto#L14-L22), and We use [Task](https://github.com/flyteorg/flyteidl/blob/cb8df7146b91b8269d1a4ea9b2a1fda42b56b6ff/protos/flyteidl/admin/task.proto#L32-L41) for get and list tasks

Reply:
awesome! thanks for explaining